src/ResourceManager/Profile/Commands.Profile/Microsoft.Azure.Commands.Profile.dll-Help.xml

```diff
@@ -33,32 +33,75 @@
 <maml:uri /></dev:type>
 <dev:defaultValue>None</dev:defaultValue>
 </command:parameter>
-<command:parameter required="false" variableLength="true" globbing="false" pipelineInput="False" position="named" aliases="none"><maml:name>ApplicationId</maml:name>
-<maml:Description><maml:para>SPN
+<command:parameter required="false" variableLength="true" globbing="false" pipelineInput="False" position="named" aliases="none"><maml:name>Environment</maml:name>
```

Comment:
We should mark this parameter as deprecated in the code of the cmdlet.  Can you add the obsolete attribute to this parameter in the cmdlet code??.  Can you put a note here that this parameter is only for backward compatibility and users shoudl use EnvironmentName instead?